packages/react-charting/docs/TestPlans/VerticalStackedBarChart/ComponentTests.md

```diff
@@ -0,0 +1,73 @@
+**Vertical Stacked Bar Chart – Component test plan**
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28594&commitId=30d8fc28ec37cd9bfff033032eb4cbcedb7db1ff&env=prod&clientType=FLUENTUI)


  <details><summary><b>react-charting-PieChart</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-PieChart.Basic.default.chromium.png|3803| Changed|
|react-charting-PieChart.Basic - RTL.default.chromium.png|3805| Changed|
|react-charting-PieChart.Basic - Dark Mode.default.chromium.png|4061| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Reply:
Conflict with master